javascript/ql/src/Security/CWE-116/IncompleteSanitization.ql

```diff
@@ -143,12 +143,26 @@ predicate whitelistedRemoval(StringReplaceCall repl) {
   )
 }
 
-from StringReplaceCall repl, DataFlow::Node old, string msg
+/**
+ * Gets a nice string representation of the pattern or value of the node.
+ */
+predicate getPatternOrValueString(DataFlow::Node node, string patternOrValue) {
```

Comment:
Predicates whose name starts with `get` usually have a result. Either rename the predicate or (better) move `patternOrValue` into the result. (This does not change anything except the syntax used for invoking the predicate).

```suggestion
string getPatternOrValueString(DataFlow::Node node) {
```
(If you apply the suggestion, also replace all occurrences of `patternOrValue` with `result`.

Reply:
Since applying suggestions one by one would of broke the build I made new commit with you co-auth 9d4e737bc280e78bb0c5c37fc834b60e93065d9a. Does this look good?

Reply:
Looks great! 👍